admin_interface/models.py

```diff
@@ -340,6 +340,9 @@ def get_active_theme():
     list_filter_sticky = models.BooleanField(
         default=True, verbose_name=_("sticky position")
     )
+    list_filter_quick_remove = models.BooleanField(
+        default=True, verbose_name=_("quick remove")
```

Comment:
Should this help text be `quick remove links` ?  `quick clear links at top of sidebar` ?

Or short text is ok if README explains it with a full sentence and a screenshot?

Reply:
I think that the right name for this option is `list_filter_chips` (chips is the technical name for this kind of control, generally used with tags).

The `help_text` could be `"quick remove active filters at top of sidebar"`.

Reply:
One issue is that `chips` is about the presentation, not the functionality.  What do you think of the compromise `list_filter_removal_chips` ?

Reply:
I like it!

Reply:
I had forgotten that I am not proposiing chips any more!

From my first comment here:
> In the end I applied simple styling instead of the «chip» idea.

This was both for a practical reason (easier to start from the existing styling on `li` and adjust than add pill styling to `a`) and a UI reason: when you see chips for example in a many-to-many autocomplete field form, you expect to be able to delete more than one in a row, because they are little informative + interactive elements, not links that refresh the whole page and make you wait.

Reply:
So... `list_filter_removal_links`?